### PR TITLE
backported radrelay fix

### PR DESCRIPTION
**Pull request by alanbuxey**

### CODE REVIEW

File: raddb/radrelay.conf.in [hunk 1]
```diff
@@ -128,39 +130,41 @@ home_server_pool radrelay {
...
-#
-#  Read the detail file.
-#
-listen {
-	type = detail
+server radrelay {
+	#
+	#  A dummy realm.
+	#
+	realm radrelay {
```
Comment (alandekok) [inline raddb/radrelay.conf.in]: realms can't go into a "server" section....

Comment (alandekok) [inline raddb/radrelay.conf.in]: Grrr... I'll go fix the v4.0.x branch

**Comment by alandekok** [comment]
I've pulled the changes over manually, including the fix.

File: raddb/radrelay.conf.in (hunk 1 above)
Comment (alanbuxey) [inline raddb/radrelay.conf.in]: doh. of course :)

**Pull request closed without merging**